android/app/src/main/java/updated/mysterium/vpn/di/Modules.kt

```diff
@@ -21,14 +22,15 @@ object Modules {
         single { NodeRepository(get()) }
         single { UseCaseProvider(get(), get()) }
 
-        viewModel { CountrySelectViewModel(get()) }
+        viewModel { SelectNodeViewModel(get()) }
         viewModel { SplashViewModel(get(), get()) }
-        viewModel { ManualConnectViewModel(get()) }
+        viewModel { HomeViewModel(get()) }
+        viewModel { FilterViewModel() }
```

Comment:
separate this lines for better read:
viewModel { 
    CountrySelectViewModel(get()) 
}

Reply:
Fix